guides/micronaut-google-cloud-compute/micronaut-google-cloud-compute.adoc

```diff
@@ -0,0 +1,123 @@
+common:header.adoc[]
+
+You will deploy a Micronaut application to https://cloud.google.com/compute/docs[Google Cloud Compute Engine], a computing and hosting service.
+
+common:costs-gcloud.adoc[]
+
+common:gcp-deploy-requirements2content.adoc[]
+
+== Google Cloud Compute
+
+=== Create the VM Instance
+
+You can create the Google Cloud Compute VM via the CLI.
+
+[source, bash]
+----
+gcloud compute instances create \
+  guide-micronaut-jdbc-compute \
+  --machine-type=e2-micro \
+  --zone=us-east1-b
+----
+
+NOTE: if you've omitted to specify a default zone in the setup steps, the command will respond requesting a zone for your instance.
+
+You will see output such as:
+
+[source, bash, options="nowrap"]
+----
+Created [https://www.googleapis.com/compute/v1/projects/micronaut-guides-jdbc/zones/us-east1-b/instances/guide-micronaut-jdbc-compute].
+NAME                          ZONE        MACHINE_TYPE  PREEMPTIBLE  INTERNAL_IP    EXTERNAL_IP     STATUS
+guide-micronaut-jdbc-compute  us-east1-b  e2-micro                   10.xxx.xxx.4   34.xxx.xxx.200  RUNNING
+----
+
+Make note of your external IP address, you will need this later.
+
+=== Open Port 8080
+
+[source, bash]
+----
+gcloud compute firewall-rules create allow-8080 --allow tcp:8080
+----
+
+You will see output such as:
+
+[source, bash, options="nowrap"]
+----
+NAME        NETWORK  DIRECTION  PRIORITY  ALLOW     DENY  DISABLED
+allow-8080  default  INGRESS    1000      tcp:8080        False
+----
+
+=== Copy the JAR to the Instance
+
+[source,bash]
+----
+gcloud compute scp build/libs/micronautguide-0.1-all.jar \
+    guide-micronaut-jdbc-compute:.
+----
+
+NOTE: If you have not set up SSH key file for `gcloud`, you will be prompted to set a passphrase to set public/private RSA key pair.
```

Comment:
```suggestion
NOTE: If you have not set up an SSH key file for `gcloud`, you will be prompted to set a passphrase to set a public/private RSA key pair.
```
The verb "set" is used three times in this sentence. Could the last one be changed to "define"? ("set a passphrase to *define* a public/private RSA key pair")